package-lock.json

```diff
@@ -31,7 +31,7 @@
         "@web3-react/injected-connector": "^6.0.7",
         "@web3-react/network-connector": "^6.1.9",
         "@web3-react/types": "^6.0.7",
-        "@web3-react/walletconnect-connector": "^6.2.10",
+        "@web3-react/walletconnect-connector": "^6.2.13",
```

Comment:
your `package-lock` should not be changing unless your `package.json` is. Please use `npm ci` instead of `npm i`/`npm install` unless you're adding a new dependency.

Reply:
Does it work when there is conflict on `package-lock.json` because I am knowing that `npm ci` command can only install with an existing package-lock.json.